src/commands/ps.rs

```diff
@@ -2,7 +2,8 @@ use crate::commands::WholeStreamCommand;
 use crate::errors::ShellError;
 use crate::object::process::process_dict;
 use crate::prelude::*;
-use sysinfo::SystemExt;
+#[allow(unused)]
+use sysinfo::{RefreshKind, SystemExt};
```

Comment:
Is `SystemExt` actually used? If not, we can probably move the `RefreshKind` import inside the `not(linux)` block and remove the `allow`.

Reply:
@twe4ked - SystemExt is used for `::new` it looks like, but you're right that I could move the RefreshKind into the block and not have to use the allow(used) here.